frontend/app-development/layout/AppBar/appBarConfig.tsx

```diff
@@ -51,6 +52,12 @@ export const menu: TopBarMenuItem[] = [
     repositoryTypes: [RepositoryType.App],
     featureFlagName: 'processEditor',
   },
+  {
+    key: TopBarMenu.PolicyEditor,
+    link: '/:org/:app/policy-editor',
+    repositoryTypes: [RepositoryType.App],
+    featureFlagName: 'policyEditor',
```

Comment:
whoo, love to see that you are using our new feature-flag utils! Kudos! 👏

Reply:
It is a very nice feature! 😄